### PR TITLE
polls: basic single question component with other text input choice a…

### PR DESCRIPTION
**Pull request by phillimorland**
…nd answers passed to state

### CODE REVIEW

**Comment by phillimorland** [comment]
@khamui this is unstyled but we can maybe merge and go on as thought you might need the states? also assuming it'll need some tweeks but not sure yet what will be needed

**Comment by khamui** [comment]
cool! yes styling can happen later. i was thinking about the state and probably its easier to send it onChange back to the parent component (PollQuestions.jsx), i have to provide you some props methods though. Similar to the EditPollQuestions.

So i keep the state updated on the parent and if user hits save we send that entire thing.

syntaxwise is cool. i mean you could convert it to functional components and get rid of the class notation :D

File: adhocracy4/polls/assets/PollQuestion.jsx [hunk 1]
```diff
@@ -94,171 +43,82 @@ class PollQuestion extends React.Component {
...
-                        type="checkbox"
+                        className="input-group__input"
+                        type="text"
                         name="question"
-                        id={'id_choice-' + choice.id + '-multiple'}
-                        value={choice.id}
-                        checked={checked}
-                        onChange={this.handleOnMultiChange.bind(this)}
+                        id={'id_choice-' + choice.id + '-single'}
+                        onChange={this.handleTextChange.bind(this)}
```
Comment (khamui) [inline adhocracy4/polls/assets/PollQuestion.jsx:112]: `{() => props.khaProvidedMethod(question.id, valueOfInput)}`

**Comment by phillimorland** [comment]
@khamui ok, second attempt, hope thats ok? i've decided to leave the commits as is becasue it will help me to see the steps in future when I have to convert other components.

File: adhocracy4/polls/assets/PollQuestion.jsx [hunk 2]
```diff
@@ -1,124 +1,72 @@
...
+              return (
+                <label className="poll-row radio" key={choice.id} htmlFor={'id_choice-' + choice.id + '-multiple'}>
+                  <input
+                    className="poll-row__radio radio__input"
+                    type="checkbox"
+                    name="question"
+                    id={'id_choice-' + choice.id + '-multiple'}
+                    value={choice.id}
+                    checked={checked}
+                    onChange={(e) => { props.OnMultiChange(props.question.id, e.target.value).bind(this) }}
```
Comment (khamui) [inline adhocracy4/polls/assets/PollQuestion.jsx:51]: think you can omit `.bind(this)` and a little typo `OnMulti..` should be `onMulti..` right?

**Pull request merged**